dataservice_publisher/resources/ready.py

```diff
@@ -0,0 +1,32 @@
+"""Repository module for ready."""
+import logging
+from os import environ as env
+from typing import Any
+
+from aiohttp import ClientConnectionError, ClientSession
+from aiohttp import web
+from dotenv import load_dotenv
+
+# Get environment
+load_dotenv()
+FUSEKI_HOST = env.get("FUSEKI_HOST", "http://fuseki")
+FUSEKI_PORT = int(env.get("FUSEKI_PORT", 8080))
+
+
+class Ready(web.View):
+    """Class representing ready resource."""
+
+    async def get(self) -> Any:
+        """Ready route function."""
+        url = f"{FUSEKI_HOST}:{FUSEKI_PORT}/fuseki/$/ping"
+        try:
+            # Get ready status from fuseki
+            async with ClientSession() as session:
+                async with session.get(url) as response:
+                    pass
+            if response.status == 200:
+                return web.Response(text="OK")
```

Comment:
Er det ikke naturlig at disse to indenteres og erstatter `pass`?

Reply:
Bra spotta. Fixa som ein del av ebaa8d6